src/jats/jats-exporter.ts

```diff
@@ -1879,7 +1858,53 @@ export class JATSExporter {
     //   }
     // }
   }
+  private appendCorrespondingToElement = (
+    corresponding: Corresponding,
+    element: HTMLElement
+  ) => {
+    const correspondingEl = this.document.createElement('corresp')
+    correspondingEl.setAttribute('id', normalizeID(corresponding._id))
+    if (corresponding.label) {
+      const labelEl = this.document.createElement('label')
+      labelEl.textContent = corresponding.label
+      correspondingEl.appendChild(labelEl)
+    }
+    correspondingEl.append(corresponding.contents)
+    element.appendChild(correspondingEl)
+  }
+
+  private getUsedCorrespondings = (contributors: Contributor[]) => {
```

Comment:
Can't we do this in a simpler way? Something like:
```
const corresps = []
contributors.forEach((c) => {
   c.corresp.flatMap((cc) => cc.correspID).map((id) => this.modelMap.get(id)).forEach(corresps.push)
})
```

Reply:
Sure